cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -2666,6 +2666,16 @@ declare namespace Deno {
      * @category File System
      */
     utimeSync(atime: number | Date, mtime: number | Date): void;
+    /**
+     * Checks if the file resource is a TTY (terminal).
+     *
+     * ```ts
+     * // This example is system and context specific
+     * using file = await Deno.open("/dev/tty6");
+     * file.isTerminal(); // true
+     * ```
+     */
+    isTerminal(): boolean;
```

Comment:
We should mark it as unstable for now so we can land it in the next patch release